recipe/meta.yaml

```
@@ -20,13 +20,14 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - dataclasses
-    - jax >=0.2.6
+    - jax >=0.3.2
     - matplotlib-base
     - msgpack-python
     - numpy >=1.12
-    - optax >=0.0.6
+    - optax
```

```suggestion
    - optax >=0.0.6
```
Why did you remove this?